instrumentation/ratpack/ratpack-1.7/library/src/main/java/io/opentelemetry/instrumentation/ratpack/client/RatpackHttpTracing.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.ratpack.client;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.instrumentation.api.instrumenter.Instrumenter;
+import ratpack.exec.ExecInitializer;
+import ratpack.http.client.HttpClient;
+import ratpack.http.client.HttpResponse;
+import ratpack.http.client.RequestSpec;
+
+/** Entrypoint for tracing OkHttp clients. */
+public final class RatpackHttpTracing {
```

Comment:
Because Ratpack HTTP client can really only be used in a Ratpack server, do you think it makes sense to just keep one package and combine the entrypoints into one? Notably `configureServerRegistry` can add the exec initializer too to keep things simpler.

Reply:
As far as we are happy to add the exec initializer at all times, I think might be easier to configure.

The only downside would be having too many classes in a package without structure while having `client` `server` and `internal` might help to clarify. 

Having different packages with a single `RatpackTracing` have the problem that a few classes would need to be `public`, which I understood was not ideal.

What do you think would be nicer? @anuraaga

Reply:
I made a refactor to have a flatter package structure